python/ray/util/annotations.py

```diff
@@ -107,22 +113,52 @@ def Deprecated(*args, **kwargs):
     if len(args) == 1 and len(kwargs) == 0 and callable(args[0]):
         return Deprecated()(args[0])
 
-    message = (
-        "**DEPRECATED:** This API is deprecated and may be removed in a future "
-        "Ray release."
+    doc_message = (
+        "\n    DEPRECATED: This API is deprecated and may be removed "
+        "in future Ray releases."
+    )
+    warning_message = (
+        "This API is deprecated and may be removed in future Ray releases. "
+        "You could suppress this warning by setting env variable "
+        "RAY_DISABLE_DEPRECATION_WARNING=0."
```

Comment:
btw I think users can also use `filterwarning`. I think this is a standard way of suppressing warnings from python (https://docs.python.org/3/library/warnings.html#warnings.filterwarnings). Maybe we can also document it?